cms/templates/js/signatory-editor.underscore

```diff
@@ -0,0 +1,33 @@
+     <div class="signatory-panel-default">
+
+       <% if (!is_editing_all_collections) { %>
+       <a class="signatory-panel-close" href="javascript:void(0);" data-tooltip="Close">
```

Comment:
@mattdrayer and @asadiqbal08, I'll take care of this quick change.

Thanks for the catches, @cptvitamin!